tests/conformance/testdata/Dockerfile.edgecases

```diff
@@ -1,3 +1,4 @@
+# Note: a registries.conf alias should redirect this to quay.io/libpod/busybox
```

Comment:
The word "should" is inherently ambiguous and should not be used in documentation (see what I did there?) In this instance, it's not clear whether you mean "we hope this happens", or "we know this will happen because there is an existing registries.conf mapping", or "we should some day set up a registries.conf mapping". This is fine to fix in the followup PR.